sql/timeseries/a11yScores.sql

```diff
@@ -1,32 +1,37 @@
 #standardSQL
-CREATE TEMPORARY FUNCTION getA11yScore(report STRING)
-RETURNS FLOAT64
+# Lighthouse changed format of scores in v3.0.0 released in July 2018 so handle old with a UDF
+CREATE TEMPORARY FUNCTION getA11yScore(reportCategories STRING)
+RETURNS FLOAT64 DETERMINISTIC
```

Comment:
With the inner query we're no longer calling this function multiple times with the same input, so if there are any memory consumption side-effects we could remove the `DETERMINISTIC`.

Reply:
Yup agreed.